statsmodels/discrete/tests/test_discrete.py

```diff
@@ -60,6 +60,46 @@
 DECIMAL_1 = 1
 DECIMAL_0 = 0
 
+
+def check_jac(self):
+    # moved from CheckModelResults
```

Comment:
why moved?  The only arg to this is `self`, seems like it should definitely be a method.